src/main/java/Duke.java

```diff
@@ -1,10 +1,99 @@
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
+        welcomeMessage();
```

Comment:
I think printWelcomeMessage() would be a better name than welcomeMessage() as it is a verb

Reply:
Oh you are right. It would be clearer to the user as well. Thanks!